### PR TITLE
docs: use `PackageManagerTabs` for `install-and-setup.mdx` code-blocks

### PR DESCRIPTION
**Pull request by m-krebs**
#### Description (required)

- update `install-and-setup.mdx` to make use of the `PackageManagerTabs` component.

#### Related issues & labels (optional)
- Suggested label: design

### CODE REVIEW

File: src/content/docs/en/install-and-setup.mdx [hunk 1]
```diff
@@ -33,26 +34,26 @@ Astro is built with Vite which targets browsers with modern JavaScript support b
...
+    <PackageManagerTabs>
+      <Fragment slot="npm">
       ```shell
       # create a new project with npm
       npm create astro@latest
       ```
-      </TabItem>
-      <TabItem label="pnpm">
+      </Fragment>
+      <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:44]:
```suggestion
      <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 2]
```diff
@@ -33,26 +34,26 @@ Astro is built with Vite which targets browsers with modern JavaScript support b
...
       ```shell
       # create a new project with pnpm
       pnpm create astro@latest
       ```
-      </TabItem>
-      <TabItem label="yarn">
+      </Fragment>
+      <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:50]:
```suggestion
      <Fragment slot="yarn">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 3]
```diff
@@ -72,35 +73,35 @@ You can also start a new astro project based on an [official example](https://gi
 
 2. Run the following command in your terminal, substituting the official Astro starter template name, or the GitHub username and repository of the theme you want to use:
 
-    <Tabs>
-      <TabItem label="npm">
+    <PackageManagerTabs>
+      <Fragment label="npm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:77]:
```suggestion
      <Fragment slot="npm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 4]
```diff
@@ -72,35 +73,35 @@ You can also start a new astro project based on an [official example](https://gi
...
       # create a new project with an official example
       npm create astro@latest -- --template <example-name>
 
       # create a new project based on a GitHub repository’s main branch
       npm create astro@latest -- --template <github-username>/<github-repo>
       ```
-      </TabItem>
-      <TabItem label="pnpm">
+      </Fragment>
+      <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:86]:
```suggestion
      <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 5]
```diff
@@ -72,35 +73,35 @@ You can also start a new astro project based on an [official example](https://gi
...
       # create a new project with an official example
       pnpm create astro@latest --template <example-name>
       
       # create a new project based on a GitHub repository’s main branch
       pnpm create astro@latest --template <github-username>/<github-repo>
       ```
-      </TabItem>
-      <TabItem label="yarn">
+      </Fragment>
+      <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:95]:
```suggestion
      <Fragment slot="yarn">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 6]
```diff
@@ -122,23 +123,23 @@ Astro comes with a built-in development server that has everything you need for
 
 Every starter template comes with a pre-configured script that will run `astro dev` for you. After navigating into your project directory, use your favorite package manager to run this command and start the Astro development server.
 
-<Tabs>
-  <TabItem label="npm">
+<PackageManagerTabs>
+  <Fragment label="npm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:127]:
```suggestion
  <Fragment slot="npm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 7]
```diff
@@ -122,23 +123,23 @@ Astro comes with a built-in development server that has everything you need for
...
   ```shell
   npm run dev
   ```
-  </TabItem>
-  <TabItem label="pnpm">
+  </Fragment>
+  <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:132]:
```suggestion
  <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 8]
```diff
@@ -122,23 +123,23 @@ Astro comes with a built-in development server that has everything you need for
...
   ```shell
   pnpm run dev
   ```
-  </TabItem>
-  <TabItem label="yarn">
+  </Fragment>
+  <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:137]:
```suggestion
  <Fragment slot="yarn">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 9]
```diff
@@ -180,23 +181,23 @@ You don't need to write TypeScript code in your Astro projects to benefit from i
 
 To check the version of your site that will be created at build time, quit the dev server (<kbd>Ctrl</kbd> + <kbd>C</kbd>) and run the appropriate build command for your package manager in your terminal:
 
-    <Tabs>
-      <TabItem label="npm">
+    <PackageManagerTabs>
+      <Fragment label="npm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:185]:
```suggestion
      <Fragment slot="npm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 10]
```diff
@@ -180,23 +181,23 @@ You don't need to write TypeScript code in your Astro projects to benefit from i
...
       ```shell
       npm run build
       ```
-      </TabItem>
-      <TabItem label="pnpm">
+      </Fragment>
+      <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:190]:
```suggestion
      <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 11]
```diff
@@ -180,23 +181,23 @@ You don't need to write TypeScript code in your Astro projects to benefit from i
...
       ```shell
       pnpm build
       ```
-      </TabItem>
-      <TabItem label="yarn">
+      </Fragment>
+      <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:195]:
```suggestion
      <Fragment slot="yarn">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 12]
```diff
@@ -268,23 +269,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
 
     Once you are in your new directory, create your project `package.json` file. This is how you will manage your project dependencies, including Astro. If you aren't familiar with this file format, run the following command to create one.
 
-    <Tabs>
-      <TabItem label="npm">
+    <PackageManagerTabs>
+      <Fragment label="npm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:273]:
```suggestion
      <Fragment slot="npm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 13]
```diff
@@ -268,23 +269,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
...
       ```shell
       npm init --yes
       ```
-      </TabItem>
-      <TabItem label="pnpm">
+      </Fragment>
+      <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:278]:
```suggestion
      <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 14]
```diff
@@ -268,23 +269,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
...
       ```shell
       pnpm init 
       ```
-      </TabItem>
-      <TabItem label="yarn">
+      </Fragment>
+      <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:283]:
```suggestion
      <Fragment slot="yarn">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 15]
```diff
@@ -294,23 +295,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
     Astro must be installed locally, not globally. Make sure you are *not* running `npm install -g astro` `pnpm add -g astro` or `yarn add global astro`.
     :::
 
-    <Tabs>
-      <TabItem label="npm">
+    <PackageManagerTabs>
+      <Fragment label="npm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:299]:
```suggestion
      <Fragment slot="npm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 16]
```diff
@@ -294,23 +295,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
...
       ```shell
       npm install astro
       ```
-      </TabItem>
-      <TabItem label="pnpm">
+      </Fragment>
+      <Fragment label="pnpm">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:304]:
```suggestion
      <Fragment slot="pnpm">
```

File: src/content/docs/en/install-and-setup.mdx [hunk 17]
```diff
@@ -294,23 +295,23 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
...
       ```shell
       pnpm add astro
       ```
-      </TabItem>
-      <TabItem label="yarn">
+      </Fragment>
+      <Fragment label="yarn">
```
Comment (ArmandPhilippot) [inline src/content/docs/en/install-and-setup.mdx:309]:
```suggestion
      <Fragment slot="yarn">
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Hi @m-krebs! Thanks for this update and welcome to team Docs! 🎉
I think it makes sense since we have a dedicated component for that. However, you forgot to update most of the `Fragment` (the `label` prop must be converted to a `slot` prop like you did for the first one).

**Comment by m-krebs** [comment]
Interestingly it works even with the label prop, so I didn't notice my mistake. But thanks for correcting :)

**Review by ArmandPhilippot — APPROVED** [review]
LGTM! Thanks again for bringing consistency in the use of components!

> it works even with the label prop

Oh weird, in the preview it didn't work... that's why I thought there was a problem!

**Review by Fryuni — APPROVED** [review]
LGTM!

Thanks for the PR and welcome to Team Docs @m-krebs 🥳🎉

And thanks for the review @ArmandPhilippot

**Review by yanthomasdev — APPROVED** [review]
Nice work, thanks!

**Pull request merged**